docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -304,6 +322,101 @@ NOTE: You must use your cluster host domain name when creating your URL.
     ✓  Successfully deleted component
 ----
 
+== Deploying a Quarkus Application on Kubernetes/Openshift cluster
+
+In this example we will be deploying a https://github.com/odo-devfiles/quarkus-ex[quarkus component] that uses GraalVM and JDK1.8+.
+
+. Download the example quarkus component. 
++
+[source,sh]
+----
+ $ git clone https://github.com/odo-devfiles/quarkus-ex && cd quarkus-ex
+----
+
+. Create an quarkus odo component 
++
+[source,sh]
+----
+   $ odo create java-quarkus myquarkus
+   Experimental mode is enabled, use at your own risk
+
+   Validation
+    ✓  Checking devfile compatibility [195728ns]
+    ✓  Creating a devfile component from registry: DefaultDevfileRegistry [170275ns]
+    ✓  Validating devfile component [281940ns]
+
+    Please use odo push command to create the component with source deployed
+----
+
+. Create a URL in order to access the deployed component:
++
+* Run below for Kubernetes cluster
++
+[source,sh]
+----
+ $ odo url create --host my-cluster.info
+  ✓  URL myquarkus-8080.my-cluster.info created for component: myquarkus
+
+ To apply the URL configuration changes, please use odo push
+----
++
+
+NOTE: You must use your cluster host domain name when creating your URL.
++
+* Run below for Openshift cluster
++
+[source,sh]
+----
+ $ odo url create 
+  ✓  URL myquarkus-8080.apps-crc.testing created for component: myquarkus
+
+ To apply the URL configuration changes, please use odo push
+----
+
+. Push the component to the cluster:
++
+[source,sh]
+----
+  $ odo push
+
+Validation
+ ✓  Validating the devfile [44008ns]
+
+Creating Kubernetes resources for component myquarkus
+ ✓  Waiting for component to start [10s]
+
+Applying URL changes
+ ✓  URLs are synced with the cluster, no changes are required.
+
+Syncing to component myquarkus
+ ✓  Checking files for pushing [951138ns]
+ ✓  Syncing files to the component [204ms]
+
+Executing devfile commands for component myquarkus
+ ✓  Executing init-compile command "mvn compile" [3m]
+ ✓  Executing dev-run command "mvn quarkus:dev" [1s]
+
+Pushing devfile component myquarkus
+ ✓  Changes successfully pushed to component
+
+----
+
+. View your deployed application in a browser using the generated url
++
+[source,sh]
+----
+ $ odo url list
+ Found the following URLs for component myspring
+ NAME              URL                                       PORT     SECURE
+ myquarkus-8080     http://myquarkus-8080.apps-crc.testing     8080     false
+----
++
+You can now continue developing your application. Just run `odo push` and refresh your browser to view the latest changes.
++
+You can also run `odo watch` to watch changes in the source code. Just refreshing the browser will render the source code changes.
++
+Run `odo delete` to delete the application from cluster.
+
 == Deploying a Java Spring Boot® component locally to Docker
```

Comment:
I would remove this whole section. Docker is not fully supported, and it might just confuse users

Reply:
Agreed with Tomas ^^ we no longer need Docker